guides/leon/vsftpd.md

```diff
@@ -0,0 +1,211 @@
+# VSFTP GUIDE
+
+## Summary 
+This is the setup guide for a vsftp box
+
+### Contact
+- Twitter: @Dubliyu
+- Slack: @yourlocalgod on wcscusf.slack.com
+- Email: [Gmail](mailto:cleonromero@mgail.com)
+
+### Table
+1. [Prerequisites](#pre-reqs)
+2. [Setup Ubuntu VM](#setup-vm)
+3. [Install VSFTPD](#install)
+4. [Secure VSFTPD](#secure)
+5. [Users](#users)
+6. [Testing](#testing)
+7. [Useful Commands](#useful)
+
+<a id="pre-reqs"></a>
+## Prerequisites 
+1. Have your [virtual environment](https://silexone.github.io/guides/nestor/ISPsetup.html) configured
+2. Have the [ISP](https://silexone.github.io/guides/nestor/ISPsetup.html) gateway running.

```

Comment:
When this is merged in along with Nestor's and Sharon's guides make another pull request with the new links. For now this is good, just a note for the future.